src/main/java/core/basesyntax/Machine.java

```diff
@@ -2,8 +2,6 @@
 
 public abstract class Machine {
     public abstract void doWork();
-

```

Comment:
The empty line 5 is needed.
You have a build failed error.
 Error:  /home/runner/work/jv-oop/jv-oop/src/main/java/core/basesyntax/Machine.java:5: 'METHOD_DEF' should be separated from previous statement. [EmptyLineSeparator]
 You can check it on local before push, just run  maven clean package from the IDE.